scripts/install-rkt.sh

```diff
@@ -29,6 +29,16 @@ tar xvzf rkt-v"${version}".tar.gz
 cat <<EOF >install-pak
 #!/bin/bash
 
+# fix mkdir issues with checkinstall and fstrans
+for dir in /usr/lib/rkt/stage1-images/\\
+        /usr/share/man/man1/\\
+        /usr/share/bash-completion/completions/\\
+        /usr/lib/tmpfiles.d/\\
+        /usr/lib/systemd/system/
+do
+    mkdir -p \$dir 2>/dev/null || :
```

Comment:
Why there is `|| :` added at the end?
If one of this directories will not be created - `checkinstall` will still have a problem and if they already existed `-p` will not fail.